shapely/geometry/base.py

```diff
@@ -351,8 +352,8 @@ def buffer(
         distance,
         resolution=16,
         quadsegs=None,
-        cap_style=CAP_STYLE.round,
-        join_style=JOIN_STYLE.round,
+        cap_style="round",
```

Comment:
> I actually asked to use "round"

Sorry, I didn't see that here.

The problem with strings is that they are out of sync with the parameter descriptions below:

```Python
cap_style : int, optional
```

Maybe changing that to 

```Python
cap_style : shapely.BufferCapStyles, optional
```

Would make it more clear that either a string that matches an enum, or the enum value, or an integer that matches an enum value all work?

Reply:
> Sorry, I didn't see that here.

Yeah, sorry, I did that in person ;) (just wanted to indicate that @akadouri didn't make that change out of the blue)

> The problem with strings is that they are out of sync with the parameter descriptions below:

That parameter description is out of date anyway, since we now do allow more with the new implementation. The top-level function has `cap_style : {'round', 'square', 'flat'}, default 'round'`, but so actually also doesn't mention the enum and is also not fully complete ..

Reply:
In updating the enums to be in line with python styles, those values now become `ROUND`, `SQUARE`, etc. I think it makes sense to direct people to using the enums, though I guess this will break backwards compatibility...

One option is to update https://github.com/shapely/shapely/blob/main/shapely/enum.py#L16 to `upper()` the input values. I'm not sure what other libraries do to allow people to avoid using enums?